docs/en/docs/tutorial/query-params.md

```diff
@@ -138,7 +138,7 @@ If you don't want to add a specific value but just make it optional, set the def
 
 But when you want to make a query parameter required, you can just not declare any default value:
 
-```Python hl_lines="6 7"
+```Python hl_lines="6  7"
```

Comment:
```suggestion
```Python hl_lines="6-7"
```

Reply:
Thanks for spotting the oversights. Just fixed em!